parameters.toml

```diff
@@ -0,0 +1,115 @@
+# General protocol parameters.
+[parameters]
+native_token = "NAAN"
+# Minimum number of blocks in an epoch.
+# This should be 10 hours given that blocks are on average 10 seconds. ( (10*60*60)/10 )
+min_num_of_blocks = 3_600
+# Maximum expected time per block (in seconds).
+max_expected_time_per_block = 10
+# Max payload size, in bytes, for a tx.
+max_tx_bytes = 1048576
+# Max payload size, in bytes, for a tx batch proposal.
+max_proposal_bytes = 6291456
+# vp allowlist
+vp_allowlist = []
+# tx allowlist
+tx_allowlist = []
+# Implicit VP WASM name
+implicit_vp = "vp_implicit"
+# Expected number of epochs per year (also sets the min duration of an epoch in seconds)
+# Epochs should be 12 hours. ( (365*24*60*60)/(12*60*60) )
+epochs_per_year = 730
+# Maximum number of signature per transaction
+max_signatures_per_transaction = 15
+# Max gas for block
+max_block_gas = 20000000
```

Comment:
With the current default value of 25_000 gas units of gas, this should allow for ~800 txs per block. Is this a reasonable amount?

Reply:
changed it to 5000000 for 200 txs per block